collect_app/src/main/res/values/magenta_theme.xml

```diff
@@ -16,6 +16,7 @@
         <item name="colorOnBackground">#000000</item>
         <item name="colorOnError">#FFFFFF</item>
         <item name="colorOnSurface">#000000</item>
+        <item name="android:textColorHint">#E3E3E3</item>
```

Comment:
I'm guessing this corrects the hint color for search? If so, I think we should have a comment here to make sure that's clear for when we're making a new theme or changing the color.

Reply:
Done.